Can't compile Godot with module_freetype_enabled=false

**Godot version:**
3.2 7b64a24eb3fcec41c5e90b18ab2f61b895efb659


**OS/device including version:**
Ubuntu 19.04


**Issue description:**
When I try to compile Godot with
```
scons p=x11 -j6 module_freetype_enabled=false verbose=yes
```
then build stops and shows this error:
```
editor/code_editor.cpp: In member function 'void CodeTextEditor::_text_editor_gui_input(const Ref<InputEvent>&)':
editor/code_editor.cpp:609:7: error: 'DynamicFont' was not declared in this scope
   Ref<DynamicFont> font = text_editor->get_font("font");
       ^~~~~~~~~~~
editor/code_editor.cpp:609:18: error: template argument 1 is invalid
   Ref<DynamicFont> font = text_editor->get_font("font");
                  ^
editor/code_editor.cpp:609:55: error: cannot convert 'Ref<Font>' to 'int' in initialization
   Ref<DynamicFont> font = text_editor->get_font("font");
                                                       ^
editor/code_editor.cpp:611:12: error: request for member 'is_valid' in 'font', which is of non-class type 'int'
   if (font.is_valid()) {
            ^~~~~~~~
editor/code_editor.cpp:612:12: error: base operand of '->' is not a pointer
    if (font->get_size() != (int)font_size) {
            ^~
editor/code_editor.cpp:613:21: error: base operand of '->' is not a pointer
     font_size = font->get_size();
                     ^~
editor/code_editor.cpp:618:40: error: base operand of '->' is not a pointer
    _add_font_size((int)font_size - font->get_size());
                                        ^~
editor/code_editor.cpp: In member function 'void CodeTextEditor::_reset_zoom()':
editor/code_editor.cpp:657:6: error: 'DynamicFont' was not declared in this scope
  Ref<DynamicFont> font = text_editor->get_font("font"); // reset source font size to default
      ^~~~~~~~~~~
editor/code_editor.cpp:657:17: error: template argument 1 is invalid
  Ref<DynamicFont> font = text_editor->get_font("font"); // reset source font size to default
                 ^
editor/code_editor.cpp:657:54: error: cannot convert 'Ref<Font>' to 'int' in initialization
  Ref<DynamicFont> font = text_editor->get_font("font"); // reset source font size to default
                                                      ^
editor/code_editor.cpp:659:11: error: request for member 'is_valid' in 'font', which is of non-class type 'int'
  if (font.is_valid()) {
           ^~~~~~~~
editor/code_editor.cpp:661:7: error: base operand of '->' is not a pointer
   font->set_size(14);
       ^~
editor/code_editor.cpp: In member function 'bool CodeTextEditor::_add_font_size(int)':
editor/code_editor.cpp:734:6: error: 'DynamicFont' was not declared in this scope
  Ref<DynamicFont> font = text_editor->get_font("font");
      ^~~~~~~~~~~
editor/code_editor.cpp:734:17: error: template argument 1 is invalid
  Ref<DynamicFont> font = text_editor->get_font("font");
                 ^
editor/code_editor.cpp:734:54: error: cannot convert 'Ref<Font>' to 'int' in initialization
  Ref<DynamicFont> font = text_editor->get_font("font");
                                                      ^
editor/code_editor.cpp:736:11: error: request for member 'is_valid' in 'font', which is of non-class type 'int'
  if (font.is_valid()) {
           ^~~~~~~~
In file included from ./core/error_macros.h:34,
                 from ./core/hash_map.h:34,
                 from ./core/ordered_hash_map.h:34,
                 from ./core/io/config_file.h:34,
                 from ./editor/editor_plugin.h:34,
                 from editor/code_editor.h:34,
                 from editor/code_editor.cpp:31:
editor/code_editor.cpp:737:28: error: base operand of '->' is not a pointer
   int new_size = CLAMP(font->get_size() + p_delta, 8 * EDSCALE, 96 * EDSCALE);
                            ^~
./core/typedefs.h:134:37: note: in definition of macro 'CLAMP'
 #define CLAMP(m_a, m_min, m_max) (((m_a) < (m_min)) ? (m_min) : (((m_a) > (m_max)) ? m_max : m_a))
                                     ^~~
editor/code_editor.cpp:737:28: error: base operand of '->' is not a pointer
   int new_size = CLAMP(font->get_size() + p_delta, 8 * EDSCALE, 96 * EDSCALE);
                            ^~
./core/typedefs.h:134:68: note: in definition of macro 'CLAMP'
 #define CLAMP(m_a, m_min, m_max) (((m_a) < (m_min)) ? (m_min) : (((m_a) > (m_max)) ? m_max : m_a))
                                                                    ^~~
editor/code_editor.cpp:737:28: error: base operand of '->' is not a pointer
   int new_size = CLAMP(font->get_size() + p_delta, 8 * EDSCALE, 96 * EDSCALE);
                            ^~
./core/typedefs.h:134:94: note: in definition of macro 'CLAMP'
 #define CLAMP(m_a, m_min, m_max) (((m_a) < (m_min)) ? (m_min) : (((m_a) > (m_max)) ? m_max : m_a))
                                                                                              ^~~
editor/code_editor.cpp:739:23: error: base operand of '->' is not a pointer
   if (new_size != font->get_size()) {
                       ^~
editor/code_editor.cpp:741:8: error: base operand of '->' is not a pointer
    font->set_size(new_size);
        ^~
editor/code_editor.cpp:748:1: error: control reaches end of non-void function [-Werror=return-type]
 }
 ^
```

--- a/SConstruct
+++ b/SConstruct
@@ -617,7 +617,7 @@ if selected_platform in platform_list:
     if env["minizip"]:
         env.Append(CPPDEFINES=["MINIZIP_ENABLED"])
 
-    editor_module_list = ["regex"]
+    editor_module_list = ["freetype", "regex"]
     if env["tools"] and not env.module_check_dependencies("tools", editor_module_list):
         print(
             "Build option 'module_"

